tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1602,10 +1600,8 @@ def test_limit_without_orderby_partial_groups_pruned(self):
         response = self.get_success_response(
             self.organization.slug,
             field=[
-                # TODO: avoid of mixing in single test release_health
-                # and performnace metrics at the same time
-                f"p50({TransactionMetricKey.MEASUREMENTS_LCP.value})",
-                f"p50({TransactionMetricKey.MEASUREMENTS_FCP.value})",
+                SessionMetricKey.HEALTHY.value,
+                SessionMetricKey.CRASH_FREE_RATE.value,
```

Comment:
Same in this test, I think its better if you just split the test into two tests; one for sessions and one for perfromance

Reply:
yeah good call.

Reply:
@ahmedetefy please check this commit 7e17da7cf5 or i need to do something completely different :)